pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
     "protobuf >= 4.23.4",
     "typing-extensions >= 4.1.0",
     "tomli >= 2.0.1; python_version < '3.11'",
+    "executing>=2.0.1",
```

Comment:
`python_version >= 3.9`? Or... optional if not using the feature?

Reply:
Started doing this but decided not to. If we accidentally eagerly import `executing` somewhere we won't notice this breaking 3.8 because it'll be installed in CI.